feat: [PL-30197]: Make entity ID more explanatory for Azure SAML use … (#963)
harness/developer-hub: docs/platform/3_Authentication/3-single-sign-on-saml.md

   ![](./static/single-sign-on-saml-81.png)

2. In **Basic SAML Configuration**, click the edit icon (pencil).

> 3. Enter a unique identifier in the **Identifier (Entity ID)** field. This can be **app.harness.io** when there is only one SAML application present for your tenant. In case of multiple SAML applications under the same tenant, this should be a unique identifier. The same identifier should be configured in the **Entity ID** field while setting up SAML on Harness side.

Reviewer: ```suggestion
3. Enter a unique identifier in the **Identifier (Entity ID)** field. When your tenant only has one SAML application, this can be `app.harness.io`. If there are several SAML applications in the same tenant, this should be a unique identifier. While setting up SAML in Harness, the same identifier should be configured in the **Entity ID** field.
```
